core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockStore.java

```diff
@@ -62,6 +63,7 @@
   private final RocksStore mRocksStore;
   private final AtomicReference<ColumnFamilyHandle> mBlockMetaColumn = new AtomicReference<>();
   private final AtomicReference<ColumnFamilyHandle> mBlockLocationsColumn = new AtomicReference<>();
+  private final LongAdder mSize = new LongAdder();
```

Comment:
Why we use `LongAdder` here.

Should we reset mSize to `0` when `clear or close` method invoked?

Reply:
LongAdder has better performance when contention is high